src/collections/Articles.ts

```diff
@@ -0,0 +1,35 @@
+import { CollectionConfig } from 'payload/types';
+
+// Example Collection - For reference only, this must be added to payload.config.ts to be used.
+const Categories: CollectionConfig = {
+  slug: 'articles',
+  access: {
+    read: () => true,
+    create: () => true,
+  },
+  fields: [
+    {
+      name: 'name',
+      type: 'text',
+      required: true,
+    },
+    {
+      name: 'price',
+      type: 'number',
+      required: true,
+    },
+    {
+      name: 'image',
+      type: 'upload',
+      relationTo: 'media',
+      required: true,
+    },
+    {
+      name: 'quantity',
+      type: 'number',
+      required: true,
+    },
```

Comment:
I think the quantity should not be related to this collection.

Reply:
Each product is created then its increase the quantity somewhere else ?

Reply:
I think a stock collection is needed where we can handle the stock